package.json

```diff
@@ -8,7 +8,6 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test": "tap test.js",
-    "prepare": "npx husky install",
```

Comment:
With husky v9 it just needs `husky`. https://typicode.github.io/husky/how-to.html#manual-setup

Reply:
Well then my PR is wrong.

Reply:
@Fdawgs fixed